projects/helper/portedTokens.js

```diff
@@ -646,6 +646,14 @@ function transformVelasAddress() {
   }
 }
 
+async function transformDfkAddress() {
+  const mapping = {
+    '0xb57b60debdb0b8172bb6316a9164bd3c695f133a': 'avax:0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7', // AVAX
```

Comment:
this might be useful for future reference by fyi we dont need it for this adapter because it prices everything in dfk tokens